packages/stores/types.ts

```diff
@@ -744,6 +744,8 @@ type TUiStore = {
     setRealAccountSignupEnd: (status: boolean) => void;
     setPurchaseState: (index: number) => void;
     simple_duration_unit: string;
+    should_show_phone_number_verified_modal: boolean;
```

Comment:
This modal will be triggered only when phone number otp is verified. Isnt?
Can we just trigger the modal on success on API call then?
do we need a observer for it

Reply:
Yeap but the modal will be displayed on personal-details-page, so when we check for verified: 1 api call this modal will show up everytime isn't 🤔 I was thinking that when user clicks on `verify` button, when the API call isSuccess then we put the value in store and redirect user back to personal details then dispaly the successful modal

Reply:
Lets check with design team on this flow